vernac/vernacexpr.ml

```diff
@@ -257,6 +257,17 @@ type vernac_argument_status = {
   implicit_status : Impargs.implicit_kind;
 }
 
+type arguments_modifier =
+  [  `Assert
+  | `ClearBidiHint
+  | `ClearImplicits
+  | `ClearScopes
+  | `DefaultImplicits
+  | `ExtraScopes
+  | `ReductionDontExposeCase
+  | `ReductionNeverUnfold
+  | `Rename ]
+
```

Comment:
What was the point to use a variant here instead of a regular sum type?

Reply:
We can see that we only produce `` `ExtraScopes | `ReductionDontExposeCase | `ReductionNeverUnfold | `Rename `` in prettyp ;)
TBH I don't know the original motivation.

Reply:
I have made a note to investigate this later.